solr/conf/protwords.txt

```diff
@@ -16,6 +16,6 @@
 
 # Some non-words that normally won't be encountered,
 # just to test that they won't be stemmed.
-dontstems
-zwhacky
+#dontstems
+#zwhacky
 

```

Comment:
Maybe we should leaves these in to test stemming? Or will we handle stemming testing later on?

Reply:
I was thinking to revisit these during MVP1 with this https://github.com/psu-libraries/psulib_blacklight/issues/139.